dusty/commands/setup.py

```diff
@@ -61,4 +62,6 @@ def save_dusty_config(config):
     for key, value in config.iteritems():
```

Comment:
Should this function be named something different?  At this point I'm not sure `save_dusty_config` is a great description for what it does